src/wallet/rpcwallet.cpp

```diff
@@ -52,6 +52,23 @@ static inline bool GetAvoidReuseFlag(CWallet * const pwallet, const UniValue& pa
     return avoid_reuse;
 }
 
+
+/* Used by RPC commands that include an include_watchonly parameter.
```

Comment:
nit: for readability perhaps employ a different verb than "include" here (have, take, etc.)

Reply:
done